Cargo.toml

```diff
@@ -33,41 +31,39 @@ ark-ff = "0.4"
 ark-poly = "0.4"
 ark-serialize = "0.4"
 ark-srs = "0.3.1"
-async-compatibility-layer = { version = "1.2.1", default-features = false, features = [
-  "logging-utils",
+async-compatibility-layer = { version = "1.1", default-features = false, features = [
+    "logging-utils",
 ] }
 async-once-cell = "0.5"
 async-std = { version = "1.12.0", features = ["attributes", "tokio1"] }
 async-trait = "0.1"
 base64 = "0.22"
 base64-bytes = "0.1"
 bincode = "1.3.3"
-bitvec = "1.0.1"
 blake3 = "1.5"
-circular-buffer = "0.1.7"
 clap = { version = "4.4", features = ["derive", "env", "string"] }
 cld = "0.5"
 derive_more = "0.99.17"
 dotenvy = "0.15"
 ethers = { version = "2.0", features = ["solc"] }
 futures = "0.3"
 
-hotshot = { git = "https://github.com/EspressoSystems/hotshot", tag = "0.5.68" }
+hotshot = { git = "https://github.com/EspressoSystems/hotshot", tag = "rc-0.5.70" }
 # Hotshot imports
-hotshot-builder-api = { git = "https://github.com/EspressoSystems/HotShot.git", tag = "0.5.68" }
-hotshot-builder-core = { git = "https://github.com/EspressoSystems/hotshot-builder-core", tag = "0.1.39" }
+hotshot-builder-api = { git = "https://github.com/EspressoSystems/HotShot.git", tag = "rc-0.5.70" }
+hotshot-builder-core = { git = "https://github.com/EspressoSystems/hotshot-builder-core", tag = "rc-0.1.40" }
 marketplace-builder-core = { git = "https://github.com/EspressoSystems/marketplace-builder-core", tag = "0.0.2" }
```

Comment:
I think marketplace-builder-core  needs to be updated to point to new hotshot tag(rc-0.5.70)

Reply:
yea, fixed all of the above in 74899c672c2ed00c1db47f4313156fe7d02dc63c